eng/performance/scenarios.proj

```diff
@@ -45,8 +45,12 @@
     <Scenario Include="New VB Console Template">
       <PayloadDirectory>$(ScenariosDir)emptyvbconsoletemplate</PayloadDirectory>
     </Scenario>
+    <Scenario Include="WinForms Large">
+      <PayloadDirectory>$(ScenariosDir)windowsformslarge</PayloadDirectory>
+    </Scenario>
   </ItemGroup>
 
+
```

Comment:
(Not here specifically) It also appears that the runs are failing due to the target framework being overwritten to just net7.0, not net7.0-windows.

Reply:
idk if targets windows is the condition we want to base on, but it should work for now at least.